web/app/themes/mitlib-parent/header.php

```diff
@@ -20,9 +19,8 @@
 	<meta charset="<?php bloginfo( 'charset' ); ?>" />
 	<!-- <meta name="format-detection" content="telephone=no"> -->
 	<!--<meta name="viewport" content="width=device-width" />-->
-	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
 	<meta name="description" content="The libraries of the Massachusetts Institute of Technology - Search, Visit, Research, Explore" />
-	<link rel="icon" type="image/png" href="<?php echo esc_attr( get_template_directory_uri() ); ?>/favicon.ico">
```

Comment:
Do we pull in a favicon elsewhere now?

Reply:
No, the site uses the stock WordPress logo as its favicon. I've created LM-189 in the backlog to add one, somehow. https://mitlibraries.atlassian.net/browse/LM-189